groth16/Cargo.toml

```diff
@@ -29,7 +29,6 @@ ff-fft = { path = "../ff-fft", default-features = false }
 r1cs-core = { path = "../r1cs-core", default-features = false }
 rand = { version = "0.7", default-features = false }
 rayon = { version = "1", optional = true }
-smallvec = "1.2.0"
```

Comment:
Did this provide any real benefit? It was only used for SmallVecs of size 2, which we'd instantly exceed after 2 `enforce` calls